### ISSUE TITLE
On-the-fly DPI changes support?

### ISSUE THREAD

**Issue by ofsjoerdk** [free-audio/clap#396]
Hi!

Playing around with reaper I found that system dpi settings are only applied when the plugin gui is first opened.
That matches the description over here https://github.com/free-audio/clap/blob/main/include/clap/ext/gui.h,
so i guess that's the way it's supposed to work.

The way vst3 does it is to notify the plugin each time the system DPI changes. For single monitor setups that's not a big deal,
but for multiple monitors with different DPI scaling on each one I think it's definately valuable. Prevents the user from
having the close and reopen the plugin gui when dragging between monitors. Would it be possible to add this feature?

**Comment by baconpaul** [comment]
I agree with @geraintluff - hosts should call set_scale whenever they need.

**Comment by geraintluff** [comment]
Vote to close, since if hosts aren't calling `.set_scale()`, that's not a CLAP standard issue?
